fetch.bs

```diff
@@ -867,7 +867,13 @@ fetch("https://victim.example/naïve-endpoint", {
     </div>
 
    <dt>`<code>range</code>`
-   <dd><p>If <var>value</var> is not a <a>simple range header value</a>, then return false.
+   <dd>
+    <ol>
+     <li><p>Let <var>rangeResult</var> be the result of <a>parse a single range header value</a>
+     given <var>value</var>.
+
+     <li><p>If <var>rangeResult</var> is failure, then return false.
+    </ol>
```

Comment:
Optional: yours is okay, but I think it's also simple enough still we could shorten it to:
```suggestion
   <dd><p>If the result of <a>parse a single range header value</a> given <var>value</var> is
   failure, then return false.
```